kolibri/plugins/slideshow_render/assets/styles/custom-hooper.css

```diff
@@ -0,0 +1,145 @@
+.hooper {
+  position: relative;
+  box-sizing: border-box;
+  width: 100%;
+  height: 200px;
+}
+.hooper * {
+  box-sizing: border-box;
+}
+.hooper-list {
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+}
+.hooper-track {
+  box-sizing: border-box;
+  display: flex;
+  width: 100%;
+  height: 100%;
+  padding: 0;
+  margin: 0;
+}
+.hooper.is-vertical .hooper-track {
+  flex-direction: column;
+  height: 200px;
+}
+.hooper.is-rtl {
+  direction: rtl;
+}
+.hooper-sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
+  border: 0;
+}
+.hooper-slide {
+  flex-shrink: 0;
+  height: 100%;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+.hooper-progress {
+  position: absolute;
+  top: 0;
+  right: 0;
+  left: 0;
+  height: 4px;
+  background-color: #efefef;
+}
+.hooper-progress-inner {
+  height: 100%;
+  background-color: #4285f4;
+  transition: 0.3s;
+}
+.hooper-pagination {
+  position: absolute;
+  right: 50%;
+  bottom: 0;
+  display: flex;
+  padding: 5px 10px;
+  transform: translateX(50%);
+}
+.hooper-indicators {
+  display: flex;
```

Comment:
we should check whether things still work without flexbox since older android browsers don't support

Reply:
@indirectlylit I imagine that it could possibly be made to work without flexbox. Non-active slides are hidden to the side and the position of the active element is placed into view. I wish I'd realized that this wasn't going to work previously I recall now having read that we can't use flex in the style guide.

I'll see what I can do to replicate the effect without flexbox but may opt to try out other OSS plugins that give similar features if I can find one that doesn't make use of flexbox.

Reply:
see a similar thread that just occurred with @bjester :)

https://github.com/learningequality/kolibri/pull/5475#discussion_r283904383

if it's too hard to change for now, let's leave it and just make sure we're aware of it and give it extra attention during integration testing

Reply:
I think I managed a solid solution. A good point from @bjester is 

>  I think in these scenarios where there is a question of whether this will work correctly, you can take into account the use and what happens when it breaks.

In my case it broke everything - Hooper was very reliant on flex to manage the spacing and flow of the slides.